src/styles/_card.scss

```diff
@@ -0,0 +1,99 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  color: $main-color;
+  font-family: $font-family;
+}
+
+.card {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+}
+
+.img {
+  display: block;
+  width: 160px;
+  height: 134px;
+}
+
+p {
+  margin: 0;
+}
+
+.product-name {
+  font-size: $regular-font-size;
+  font-weight: $regular-font-weight;
+  line-height: $regular-line-height;
+  margin-top: 36px;
+}
+
+.product-code {
+  color: $secondary-color;
+  font-size: $small-font-size;
+  font-weight: $small-font-weight;
+  line-height: 14px;
+  margin-top: 4px;
+}
+
+.product-evaluation {
+  margin: 16px 0 24px 2px;
+}
+
+.product-evaluation,
+.product-price {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.product-reviews {
+  font-size: $small-font-size;
+  font-weight: $small-font-weight;
+}
+
+.product-price {
+  height: 17px;
```

Comment:
in this case mentor told about `height` property. It shouldn't be hardcoded. Instead of this use right font-family and font-weight and then the text will look like in the layout. This is necessary in order for you to understand the basic rules and learn how to correctly define styles, imagine a situation when you have a lot of text and you set the height to 200 pixels, and then you open your page on a mobile phone and the text that began to move automatically no longer fits into the frame, because it does not have a dynamic but a static set height of 200 pixels. Therefore, everything should be as dynamic as possible
![image](https://github.com/mate-academy/layout_product-cards/assets/70009205/e30b2554-e7e0-4e57-beab-8db0a44a1d0c)